modules/Blockchain/Starfleet/contracts/ProfileStorage.sol

```diff
@@ -115,7 +114,8 @@ contract ProfileStorage {
 
     function transferTokens(address wallet, uint256 amount)
     public onlyContracts {
-        ERC20 token = ERC20(hub.tokenAddress());
-        token.transfer(wallet, amount);
+         wallet.transfer(amount);
     }
+
+    function () public payable {}
```

Comment:
Is this left here on purpose?

Reply:
Yes, a contract must have this function to be able to receive native currency